### ISSUE TITLE
new game - SailorZombie

### ISSUE THREAD

**Issue by cmjdbb** [argonlefou/DemulShooter#115]
Hi, argonlefou,there's a new game called SailorZombie. It's not yet available. Can you add a light gun to it

**Comment by faluki1983** [comment]
I join the question to see if this game could be supported

**Comment by argonlefou** [comment]
> Hi, argonlefou,there's a new game called SailorZombie. It's not yet available. Can you add a light gun to it

Hello,

I guess you're talking about the Namco SYSTEM 357 arcade game ? Which has been "leaked" recently with a so called "working version" of the Emulator ?

Here are some words from the man behind this emulator (and it's only from 2 days ago) :   

> My team and I are extremely upset that the leaks have occurred and we advise not to use the fork as it could potentially have malware (EDIT: the leaked emulator DOES indeed have malware, a keystroker, so if you have it installed, delete it IMMEDIATELY! You are at risk!). It's also a beta build so a lot of the new stuff (in which I will discuss later in the post) have not been implemented. 

[Full Source](https://www.arcade-projects.com/threads/namco-system-357-emulation-maybe-project-omed-2023.23787/post-370071)

So, first : the emulator having no public release (not even a beta release) - and same thing for the dump - there is no way to really run and play that game. So no way to add any lightgun support to anything.
Just as a reminder : PS3/System 357 arcade games are not "windows" games and can't be run on a computer without any emulator

And then : even when the release will become a true thing, there are people working on this emulator. 
So, as for all other in-dev softwares : you better ask directly gun support to them ! They know their product far better than I ever could, and native integration will be way better than dirty hack. And more respectfull to their work.

I can either help or share code if it can help them, but DemulShooter can't support that kind of software with daily (orr weekly) builds, as a new build will break compatibility and need a rewrite on my side.

In a nutshell : we just need to be patient and wait for people to finish their work, properly, and share it as they want to, when they want to.
